### PR TITLE
refactor: add default policies yaml

### PR DESCRIPTION
**Pull request by royhadad**

### CODE REVIEW

File: pkg/defaultPolicies/defaultPolicies.yaml [hunk 1]
```diff
@@ -0,0 +1,370 @@
+apiVersion: v1
```
Comment (royhadad) [inline pkg/defaultPolicies/defaultPolicies.yaml:1]: I just created a new user and then clicked the "export" button

File: bl/policy/policy_factory.go [hunk 2]
```diff
@@ -21,47 +21,53 @@ type RuleWithSchema struct {
...
 }
 
-func CreatePolicy(policies *cliClient.EvaluationPrerunPolicies, policyName string, registrationURL string, defaultRules *defaultRules.DefaultRulesDefinitions) (Policy, error) {
-	var err error
-
-	if policies == nil && policyName != "" && policyName != "Default" {
-		return Policy{}, fmt.Errorf("policy %s doesn't exist, sign in to the dashboard to customize your policies: %s", policyName, registrationURL)
+func CreatePolicy(policies *defaultPolicies.EvaluationPrerunPolicies, policyName string, registrationURL string, defaultRules *defaultRules.DefaultRulesDefinitions, isAnonymous bool) (Policy, error) {
+	if policies == nil {
+		// policies should never be nil because of the fallback of defaultPolicies.yaml
```
Comment (shmu3l) [inline bl/policy/policy_factory.go:26]: good comment

File: pkg/defaultPolicies/defaultPolicies.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import (
+	_ "embed"
+
+	yamlConvertor "github.com/ghodss/yaml"
+)
+
+//go:embed defaultPolicies.yaml
+var defaultPoliciesFileContent string
+
+func GetDefaultPoliciesYamlContent() string {
```
Comment (shmu3l) [inline pkg/defaultPolicies/defaultPolicies.go]: remove

File: pkg/defaultPolicies/defaultPolicies.go [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+func GetDefaultPoliciesStruct() *EvaluationPrerunPolicies {
+	var defaultPolicies EvaluationPrerunPolicies
+	err := yamlConvertor.Unmarshal([]byte(GetDefaultPoliciesYamlContent()), &defaultPolicies)
+	if err != nil {
+		panic(err)
+	}
+	return &defaultPolicies
+}
+
+type EvaluationPrerunPolicies struct {
```
Comment (shmu3l) [inline pkg/defaultPolicies/defaultPolicies.go:9]: move types to the top of the file

**Review by shmu3l — DISMISSED** [review]

**Review by shmu3l — DISMISSED** [review]

**Review by shmu3l — APPROVED** [review]
Great work!

**Pull request merged**